src/common/vitals/cumulative-layout-shift.js

```diff
@@ -1,13 +1,19 @@
-import { onCLS } from 'web-vitals'
+import { onCLS } from 'web-vitals/attribution'
 import { VITAL_NAMES } from './constants'
 import { VitalMetric } from './vital-metric'
 import { isBrowserScope } from '../constants/runtime'
 
 export const cumulativeLayoutShift = new VitalMetric(VITAL_NAMES.CUMULATIVE_LAYOUT_SHIFT, (x) => x)
 
 if (isBrowserScope) {
-  onCLS(({ value, entries }) => {
-    if (cumulativeLayoutShift.roundingMethod(value) === cumulativeLayoutShift.current.value) return
-    cumulativeLayoutShift.update({ value, entries })
+  onCLS(({ value, attribution, id }) => {
+    const attrs = {
```

Comment:
This is removing a logic check. Was that no longer needed or is there something else we need to put here to prevent duplicate reporting?

Reply:
The check's purpose was to not report on this callback trigger if `value` remains the same, like: [0.1, 0.2, 0.2] <- don't capture 2nd `0.2`. 
This is likely related to `reportAllChanges` being true.
However, web vitals already does not report if the value remains unchanged so we don't need this.